packaging/dpkg/build-package.sh

```diff
@@ -15,14 +15,17 @@ else
     ICE_VERSION=$(dpkg-parsechangelog --file /workspace/build/debian/changelog --show-field Version)
 fi
 
-# Generate a tarball of the current repository state for the given ICE_VERSION
-echo "Creating tarball for ICE_VERSION=$ICE_VERSION"
+UPSTREAM_VERSION=$(echo $ICE_VERSION | cut -f1 -d'-')
```

Comment:
dpkg-buildpackage expect the archive to use the upstream version (3.8.0) and not the full version in the changelog.